osquery/sql/sqlite_encoding.cpp

```diff
@@ -31,29 +37,47 @@ static void b64SqliteValue(sqlite3_context* ctx,
 
   std::string input((char*)sqlite3_value_text(argv[0]));
   std::string result;
-  if (encode) {
-    result = base64Encode(input);
-  } else {
+  if (encode & B64_ENCODE) {
+    if ((encode & B64_ENCODE_CONDITIONAL) && !isPrintable(input)) {
+      result = base64Encode(input);
+    } else {
+      result = input;
```

Comment:
Maybe I'm just misunderstanding how these fancy functions are being used, but if `B64_ENCODE` is set, wouldn't the expectation be that this is `base64Encode`d?